app/models/review.rb

```diff
@@ -11,7 +11,8 @@ class Review < ActiveRecord::Base
   validates :user, presence: true
   validates :beer, presence: true
 
-  def owner?(logged_in_user)
-    user == logged_in_user
+  def owner_or_admin?(logged_in_user)
+    return false unless logged_in_user
+    user == logged_in_user || "admin" == logged_in_user.role
```

Comment:
This makes sense but is confusing to read. Can you refactor it so it's more clear?